site/docs/capabilities/llm-integrations/connect-providers.md

```diff
@@ -0,0 +1,507 @@
+---
+id: connect-providers
+title: Connecting to AI Providers
+sidebar_position: 10
+---
+
+# Connecting to AI Providers
+
+Envoy AI Gateway provides a unified interface for connecting to multiple AI providers through a standardized configuration approach. This page explains the fundamental concepts, resources, and relationships required to establish connectivity with any supported AI provider.
+
+## Overview
+
+Establishing connectivity with an AI provider involves configuring three key Kubernetes resources that work together to enable secure, scalable access to AI services:
+
+1. **AIServiceBackend** - Defines the backend service and its API schema
+2. **BackendSecurityPolicy** - Configures authentication credentials
+3. **AIGatewayRoute** - Routes client requests to the appropriate backends
+
+These resources provide a consistent configuration model regardless of which AI provider you're connecting to, whether it's OpenAI, AWS Bedrock, Azure OpenAI, or any other [supported provider](./supported-providers).
+
+## Core Resources for Provider Connectivity
+
+### AIServiceBackend
+
+The `AIServiceBackend` resource represents an individual AI service backend and serves as the bridge between your gateway and the AI provider's API.
+
+#### Purpose and Configuration
+
+- **API Schema Definition**: Specifies which API format the backend expects (OpenAI v1, AWS Bedrock, Azure OpenAI, etc.)
+- **Backend Reference**: Points to the Envoy Gateway Backend resource
+- **Security Integration**: Links to authentication policies for upstream services
+
+#### Key Fields
+
+```yaml
+apiVersion: aigateway.envoyproxy.io/v1alpha1
+kind: AIServiceBackend
+metadata:
+  name: my-provider-backend
+spec:
+  # API schema the backend expects
+  schema:
+    name: OpenAI           # Provider API format
+    version: "v1"          # API version (optional for OpenAI, defaults to "v1")
+
+  # Reference to the Envoy Gateway Backend resource
+  backendRef:
+    name: my-provider-backend
+    kind: Backend
+    group: gateway.envoyproxy.io
+
+  # Security policy for authentication
+  backendSecurityPolicyRef:
+    name: my-provider-auth
+    kind: BackendSecurityPolicy
+    group: aigateway.envoyproxy.io
+```
+
+#### Schema Configuration Examples
+
+Different providers require different schema configurations:
+
+| Provider | Schema Configuration |
+|----------|---------------------|
+| OpenAI | `{"name":"OpenAI","version":"v1"}` |
+| AWS Bedrock | `{"name":"AWSBedrock"}` |
+| Azure OpenAI | `{"name":"AzureOpenAI","version":"2025-01-01-preview"}` |
+| GCP Vertex AI | `{"name":"GCPVertexAI"}` |
+| GCP Anthropic | `{"name":"GCPAnthropic"}` |
```

Comment:
those two are work in progress so let's wait

Reply:
well, i think we can go ahead and merge assuming they are anyways blockers fro v0.3 release 😉